core-bundle/src/Resources/contao/library/Contao/Controller.php

```diff
@@ -1384,7 +1384,7 @@ protected function redirectToFrontendPage($intPage, $strArticle=null, $blnReturn
 	protected function getParentEntries($strTable, $intId)
 	{
 		// No parent table
-		if (!isset($GLOBALS['TL_DCA'][$strTable]['config']['ptable']))
+		if (!isset($GLOBALS['TL_DCA'][$strTable]['config']['ptable']) || empty($GLOBALS['TL_DCA'][$strTable]['config']['ptable']))
```

Comment:
```suggestion
		if (empty($GLOBALS['TL_DCA'][$strTable]['config']['ptable']))
```

I'm pretty sure that `empty()` implies the `isset()` check.